Lib/pathlib.py

```diff
@@ -709,11 +709,7 @@ def __new__(cls, *args, **kwargs):
             warnings._deprecated("pathlib.PurePath(**kwargs)", msg, remove=(3, 14))
         if cls is Path:
             cls = WindowsPath if os.name == 'nt' else PosixPath
```

Comment:
Could this condition also be calculated and stored at import time? You could have a global or class `_DEFAULT_PATH` variable, and then this condition could be changed to:

```python
if cls is Path:
    cls = _DEFAULT_PATH
```

Reply:
Though this makes sense in theory, I can't seem to measure any significant performance impact from making this change! I might leave it, to keep the code a little simpler?